test/ngResource/resourceSpec.js

```diff
@@ -1052,7 +1052,8 @@ describe("basic usage", function() {
     it('should call the error callback if provided on non 2xx response', function() {
       $httpBackend.expect('GET', '/CreditCard/123').respond(ERROR_CODE, ERROR_RESPONSE);
 
-      CreditCard.get({id:123}, callback, errorCB);
+      var ccs = CreditCard.get({id:123}, callback, errorCB);
```

Comment:
I agree, but the alternative would be
https://github.com/lgalfaso/angular.js/commit/9f7ed380da3491b2a353f67f41c3f92774713b49

Do you think the alternative is best?